README.md

```diff
@@ -1,9 +1,84 @@
-Project Proposal: https://docs.google.com/document/d/18x-RVWZ8EyQLAe7ZrwN4HEShcfQ2A2eoOs4oNtzYU9Y/edit?usp=sharing
-
-# Proposal Contributions:
-- Cover Page:                   Kimleng Hor
-- Project Overview:             Kimleng Hor / Julia Chin
-- System Environment:           Dominic Pham
-- Functional Requirements:      Julia Chin / Dominic Pham
-- Non-functional Requirements:  Julia Chin / Dominic Pham
+# Delight
+# CS 157A – Team 18 Project Requirement Document
+
+## Professor:
+Dr. Ching-Seh (Mike) Wu
+
+## Team Members:
++ Julia Chin
++ Kimleng Hor
++ Dominic Pham
+
+### Project Overview
+Our team will be creating a website to help people find recipes for maintaining a healthy diet. Researchers have found that Americans eat out very often as opposed to cooking food at home. Hence, obesity is a problem in America. According to Vox, foods at restaurants have 20 to 40 percent more calories than home-cooked food. Our team wants to introduce a platform where people can choose their diets and be provided with recipes that suit their diets. With our platform, people will be able to learn how to eat right. Our stakeholders are people who are looking to learn how to maintain a healthy diet. This platform is important because it provides resources to guide people to living a healthy lifestyle.
+
+# System Environment
+
+### Structure Diagram (3-tier Architecture)
+<img src="/images/StructureDiagram.png" width=400>
+
+### Hardware and Software Used
+- Mac OSX
+- Node.js
+
+### RDBMS
+- MySQL Community Server 8.0.17
+
+### Application Languages/Frameworks
+- Node.js, React, Express, HTML, CSS, Javascript, SQL
+
+### Functional Requirements
+
+1. Create an account
+- The website will provide a form for users to fill in their information. The form requires first name, last name, email address, password and confirm password.
+- Users must provide complete information in order to proceed with the operation.
+- The system will check whether the information that is provided by the user existing in the database. 
+
+2. Manage account.
+- Users will be able to change their information; such as email address, password but not their username.
+- Users will be able to change their food preferences.
+- The system will save and remember all the information that are changed by the user.
+
+3. Food Preference
+- Users will be able to set their own preferences for food recipes; such as the level of spiciness, the food cultures, and many more.
+- The system will save the information and organize the pages to fit the user preference.
+
+4. Users will be able to filter food recipes by:
+    a. Popularity
+    b. Cooking difficulty
+    c. Spiciness
+    d. Diet (e.g. Weight Loss, Muscle Building, Vegan, Ketogenic, etc)
+    e. Type of meal (e.g. breakfast, lunch, dinner, snack, dessert, drink)
+    f. Type of cuisine (e.g. American, Asian, Mexican, etc)
+    g. Amount of time to cook meal
+    
+5. Users will be able to use a search bar to easily navigate recipes.
```

Comment:
Instead of having the whole sentence, we can just put "search bar", and do the same thing for number 7.